src/clients/python/experimental_api_v2/library/grpcclient.py

```diff
@@ -635,102 +661,93 @@ def wrapped_callback(call_future):
                 error = get_error_grpc(rpc_error)
             callback(result=result, error=error)
 
-        request = self._get_inference_request(inputs, outputs, model_name,
-                                              model_version, request_id,
-                                              parameters)
+        request = _get_inference_request(inputs, model_name, model_version,
+                                         outputs, request_id)
 
         try:
             self._call_future = self._client_stub.ModelInfer.future(request)
             self._call_future.add_done_callback(wrapped_callback)
         except grpc.RpcError as rpc_error:
             raise_error_grpc(rpc_error)
 
-    def _get_inference_request(self, inputs, outputs, model_name, model_version,
-                               request_id, parameters):
-        """Creates and initializes an inference request.
+    def async_sequence_infer(self,
+                             sequence_metadata,
```

Comment:
Just "sequence". Should the response_pool just be associated with the sequence? I think use_streaming should be set in the sequence.